src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -3820,6 +3831,20 @@ def set_up_extended_location(self, mc: ManagedCluster) -> ManagedCluster:
             )
         return mc
 
+    def set_up_disable_local_accounts(self, mc: ManagedCluster) -> ManagedCluster:
```

Comment:
If the newly added option `disable_local_accounts` is not related to other options, is not part of a complex profile and does not need to be set up separately to avoid confusion with other logic, I suppose you could omit this function (`set_up_disable_local_accounts`) and set it to the `ManagedCluster` object directly in `init_mc` (and don't forget to update the docstring/comments 😃).